pkg/scheduler/framework/plugins/nodeaffinity/node_affinity_test.go

```diff
@@ -45,15 +45,18 @@ func TestNodeAffinity(t *testing.T) {
 		disablePreFilter    bool
 	}{
 		{
-			name: "no selector",
-			pod:  &v1.Pod{},
+			name:                "no selector and affinity",
```

Comment:
how is this different from test case "Pod with no Affinity will schedule onto a node" below?